addon/-private/system/model/errors.js

```diff
@@ -117,7 +98,6 @@ export default ArrayProxy.extend(Evented, {
     this.on('becameValid', target, becameValid);
   },
```

Comment:
cc @stefanpenner tests broke when I tried to remove `_registerHandlers`, suspect it's just some tests that need updating but would love you to poke at this if you have a moment.